test/todo.js

```diff
@@ -0,0 +1,40 @@
+var tap = require('tap');
+var tape = require('../');
+var concat = require('concat-stream');
+
+tap.test('tape todo test', function (assert) {
+    var test = tape.createHarness({ exit: false });
+    assert.plan(1);
+
+    test.createStream().pipe(concat(function (body) {
+        assert.equal(
+            body.toString('utf8'),
+            'TAP version 13\n'
+            + '# success\n'
+            + 'ok 1 should be equal\n'
+            + '# failure\n'
+            + 'not ok 2 should be equal # TODO\n'
+            + '  ---\n'
+            + '    operator: equal\n'
+            + '    expected: false\n'
+            + '    actual:   true\n'
+            + '  ...\n'
+            + '\n'
+            + '1..2\n'
+            + '# tests 2\n'
+            + '# pass  2\n'
+            + '\n'
+            + '# ok\n'
+        )
+    }));
+
+    test("success", function (t) {
```

Comment:
Happily :) I think I saw some test names with double quotes elsewhere so kinda went with that. Wasn't really sure what was expected tbh.